modules/atomic-widgets/module.php

```diff
@@ -77,6 +79,9 @@ public function __construct() {
 			add_filter( 'elementor/widgets/register', fn( Widgets_Manager $widgets_manager ) => $this->register_widgets( $widgets_manager ) );
 			add_action( 'elementor/atomic-widgets/settings/transformers/register', fn ( $transformers ) => $this->register_settings_transformers( $transformers ) );
 			add_action( 'elementor/atomic-widgets/styles/transformers/register', fn ( $transformers ) => $this->register_styles_transformers( $transformers ) );
+			add_action( 'elementor/elements/elements_registered', function ( $elements_manager ) {
```

Comment:
```suggestion
			add_action( 'elementor/elements/elements_registered', fn ( $elements_manager ) {
```
Maybe? Just for consistency.